src/routes.py

```diff
@@ -111,6 +111,37 @@ def device_create():
 
     return jsonify(res), 200
 
+@api.route('device/edit', methods=["PATCH"])
+def device_edit():
+
+    device_data = request.get_json()
+    ID_device = device_data["DeviceID"]
+    new_area = float(device_data["NewArea"])
+
+    devices_dao.update_area(ID_device, new_area)
+
+    res = {
+        'code': 'success',
+        'description': 'device edited'
+    }
+
+    return jsonify(res), 200
+
+
+@api.route('device/delete', methods=["DELETE"])
+def device_delete():
+    device_data = request.get_json()
+    ID_device = device_data["DeviceID"]
+
+    devices_dao.delete_device(ID_device)
```

Comment:
E aqui precisamos de um for para deletar cada ID. Depois melhoro isso com um método DAO que recebe uma lista de IDs.